### PR TITLE
Add description for each feature in the review block

### LINKED ISSUE #1260: Review Block: Add descriptions to product ratings
### Description:
It would be nice to have an option to display the reasons for the rating for each of the product features. 
Something like: https://prnt.sc/QYWb9sO-vIkD

### Alternatives:
n/a

### PR DESCRIPTION
**Pull request by arinaturcu**
Closes #1260.

### Summary

Add a description for each feature of the product in the Review Block, under the review.

### Screenshots 
Screenshot 2023-01-04 at 20 15 32

----

### Test instructions

- In the `Product Features` panel, when a feature is clicked, the text area for the description should appear under the `rating` control.
- It will inherit the text color, and will have `0.7 opacity` and `80% font size` of the feature title font size (which is the content font size)
- we need to make sure it looks ok with both styles (One Column and Two Columns)

---- 

### Checklist before the final review

- [ ] Visual elements are not affected by independent changes.
- [x] It is at least compatible with the [minimum WordPress version](https://wordpress.org/plugins/otter-blocks/).
- [x] It loads additional script in frontend only if it is required.
- [x] Does not impact the [Core Web Vitals](https://web.dev/vitals/).
- [x] In case of deprecation, old blocks are safely migrated.
- [x] It is usable in Widgets and FSE.
- [x] Copy/Paste is working if the attributes are modified.

### CODE REVIEW

File: src/blocks/blocks/review/block.json [hunk 1]
```diff
@@ -35,19 +35,23 @@
...
 				"title": "Look & Feel",
-				"rating": 9
+				"rating": 9,
+				"description": "Beautiful colours"
 			},
 			{
 				"title": "Price",
-				"rating": 7
+				"rating": 7,
+				"description": "Good price for the quality"
```
Comment (HardeepAsrani) [inline src/blocks/blocks/review/block.json:54]: You don't need to add them by default.

File: src/blocks/blocks/review/edit.js [hunk 2]
```diff
@@ -341,6 +341,14 @@ const Edit = ({
 											{ /** translators: %s Rating score. */ sprintf( __( '%f out of %f', 'otter-blocks' ), Math.max( Math.abs( feature.rating / divide ).toFixed( 1 ) || 0, 1 ), 10 / divide ) }
 										</span>
 									</div>
+
```
Comment (HardeepAsrani) [inline src/blocks/blocks/review/edit.js:344]: You can make sure it doesn't appear if the block isn't selected.

**Review by HardeepAsrani — CHANGES_REQUESTED** [review]
We can remove the description by default and also make sure it doesn't appear in editor if it's empty and the block isn't selected.

**Review by HardeepAsrani — APPROVED** [review]

**Comment by irinelenache** [comment]
@arinaturcu Tested and everything works fine on my end 🚀

**Pull request merged**